src/modules/AggregateRepository.js

```diff
@@ -98,17 +97,18 @@ export default class AggregateRepository {
                 return Promise.reject(error);
               }
             )
-
         }
       });
   }
 
   createEntity({ EntityClass, command, options }) {
-
-    const entity = new EntityClass();
-
+    let entity;
```

Comment:
This code 105-110 is duplicated and should be extracted into a function

Reply:
Resolved with: https://github.com/eventuate-clients/eventuate-client-nodejs/pull/45/commits/726e8d6663662ee029d2e21c1932787feaa962fd